src/frontend/src/contractContex.ts

```diff
@@ -2,10 +2,12 @@ import { createContext, useContext, Dispatch } from "react";
 import Contract from './models/contract';
 import { CollectionAction } from './reducers/utils';
 import { ContractState } from './reducers/contractReducer';
-
+import PersonnelRequest from './models/PersonnelRequest';
 export interface IContractContext {
     contract: Contract | null;
     isFetchingContract: boolean;
+    editRequests: PersonnelRequest[] | null;
+    setEditRequests: (request: PersonnelRequest[] | null) => void;
```

Comment:
Do `editRequests` and `setEditRequests` belong on the contract context? Or should they be scoped to the request sidesheet?